cmd/create.go

```diff
@@ -60,10 +65,30 @@ var createCmd = &cobra.Command{
 		}
 
 		sendCompleteInstallTelemetry(globalFlags.DryRun, globalFlags.UseTelemetry)
-		reports.HandoffScreen(globalFlags.DryRun, globalFlags.SilentMode)
-		time.Sleep(time.Millisecond * 2000)
 		log.Println("Kubefirst installation finished successfully")
 		informUser("Kubefirst installation finished successfully", globalFlags.SilentMode)
+		log.Println("Skiping the presentation of console and api for the handoff screen")
+		if createFlags.EnableConsole {
+			log.Println("Starting the presentation of console and api for the handoff screen")
+			go func() {
```

Comment:
the goroutines are missing the join point

Reply:
What I am missing here?

On this use case, I am using the handoof screen to keep this routines alive. Both create a server that never returns unless there is a panic. 

Once we we close the handoff screen both as destroyed as desired. 

Do I really need the join point? What that helps on that scenario? I am may be missing something, but I am not clear on what.

Reply:
I disable this until we reach agreement on the feature, although it works as design. I need to merge this PR to get the other fix it has to unblock github install tests.